src/App.js

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import Pokemon from "./components /Pokemon";
+import pokemonlogo from "./assets/International_Pokémon_logo.svg.png"
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemonNames, setPokemonNames] = useState({});
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
    const [pokemonNames, setPokemonNames] = useState();
```